eosiojava/src/main/java/one/block/eosiojava/session/TransactionProcessor.java

```diff
@@ -399,7 +437,7 @@ public boolean sign() throws TransactionSignError {
         EosioTransactionSignatureRequest eosioTransactionSignatureRequest;
         try {
             eosioTransactionSignatureRequest = this.createSignatureRequest();
-        } catch (TransactionCreateSignatureRequestError transactionCreateSignatureRequestError) {
+        } catch (TransactionCreateSignatureRequestError | SerializeContextFreeDataError transactionCreateSignatureRequestError) {
```

Comment:
Maybe catching the new error in a separate catch is better. That also could give the new error a separate ErrorContants which could help debugging easier.

Reply:
The error is already thrown at https://github.com/EOSIO/eosio-java/pull/115/files#diff-920c63470d3ab58ee720f4ab72ce2138R935, so you can tell where the exception occurred in the console. I can split them out to separate `catch`s, but it's going to basically be the same thing since this is an upstream catch.
Added error to `serializeContextFreeData` call: `throw new SerializeContextFreeDataError("Something happened in serializeContextFreeData");`

Stacktrace:
<img width="1091" alt="Screen Shot 2020-05-08 at 3 19 18 PM" src="https://user-images.githubusercontent.com/11666182/81441286-847d6900-913f-11ea-993e-6a7a1965b2ff.png">